### PR TITLE
[macos] hack around macos test problem

### PR DESCRIPTION
**Pull request by derekwbrown**
### What does this PR do?

Two previous PRs inadvertently broke the MacOS tests, in spite of the fact that the related code
doesn't ship on MacOS.

This PR provides a temporary workaround to allow the MacOS tests to pass, until a better solution can be found

### Motivation

Passing tests.

### Reviewer's Checklist

- [x] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [x] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [x] Adequate QA/testing plan information is provided if the `qa/skip-qa` label is not applied.
- [ ] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the `k8s/<min-version>` label, indicating the lowest Kubernetes version compatible with this feature. 
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

File: pkg/security/probe/probe_unsupported.go [hunk 1]
```diff
@@ -0,0 +1,15 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2022-present Datadog, Inc.
+// +build linux
```
Comment (brycekahle) [inline pkg/security/probe/probe_unsupported.go:5]: There is an extra `// +build` comment here

Comment (derekwbrown) [reply pkg/security/probe/probe_unsupported.go:5]: Yes.  that is an artifact of a bug in the `go generate` stage.  it either puts it there, or before the copyright, which in turn fails the copyright linting.

I have raised this issue with CWS for a future PR, but objective is to have the actual tests pass.

Comment (brycekahle) [reply pkg/security/probe/probe_unsupported.go:5]: Where is this file being generated from?

Comment (derekwbrown) [reply pkg/security/probe/probe_unsupported.go:5]: `pkg/security/secl`

**Review by paulcacheux — APPROVED** [review]

**Review by truthbk — APPROVED** [review]

File: pkg/security/probe/probe_unsupported.go (hunk 1 above)
Comment (paulcacheux) [reply pkg/security/probe/probe_unsupported.go:5]: I'm going to take a look at what is generating this

Comment (paulcacheux) [reply pkg/security/probe/probe_unsupported.go:5]: This is NOT an artifact of a bug in the `go generate` stage, this file is not even generated

Comment (derekwbrown) [reply pkg/security/probe/probe_unsupported.go:5]: sorry.  You're right.  this one I think is copy paste error.  @paulcacheux

**Pull request merged**